.eslintrc.js

```diff
@@ -70,6 +71,7 @@ module.exports = {
 
     'import/extensions': 'off',
     'import/prefer-default-export': 'off',
+    'import/no-extraneous-dependencies': ['error', {'devDependencies': true, 'optionalDependencies': false, 'peerDependencies': false}],
```

Comment:
according to https://github.com/import-js/eslint-plugin-import/blob/main/docs/rules/no-extraneous-dependencies.md , `devDependencies:false` allows dev dependencies to be imported. We want that to be an error for non-test code.

To set this only for test code, put it in the `overrides.rules` (line 20 above)?

Reply:
Fixed